source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -40,13 +36,12 @@ Before you install any new packages, update |CL| with the following command:
 Install Kubernetes and CRI runtimes
 ***********************************
 
-Kubernetes and a set of supported :abbr:`CRI (Container Runtime Interface)`
-runtimes are included in the `cloud-native-basic`_ bundle. To install the
-framework, enter the following command:
+Kubernetes, a set of supported :abbr:`CRI (Container Runtime Interface)`
+runtimes, and networking plugins, are included in the `cloud-native-basic`_ and `containers-basic`_ bundles. To install this framework, enter the following command:
```

Comment:
do we need `containers-basic` to run crio and kata? the recent change in bundles seems to have included all the bits into `cloud-native-basic` itself.

Reply:
containers basic is needed only for weave. Canal and Calico work with just cloud-native-basic

Reply:
@ganeshmaharaj , @mcastelino. I've removed `containers-basic` from this tutorial. For now, this tutorial only has to show it works with `flannel`.  In case you're not aware, I'm trying to expedite this PR because the Deep Learning tutorial, PR 334, depends on it, and on its use of `flannel`. 

I will expect to see changes to CL bundle for `Weave` next week--so I won't include it in this PR. I can incorporate bundle changes next week per @bryteise.

Reply:
+1. we can always make changes later. this round need not cover all the things.